setup-tdx-guest.sh

```diff
@@ -51,6 +51,3 @@ grub_set_kernel
 if [[ "$KERNEL_RELEASE" == *-generic ]]; then
   apt install --yes "linux-modules-extra-${KERNEL_RELEASE}"
 fi
-
-# setup attestation
-"${SCRIPT_DIR}"/attestation/setup-guest.sh
```

Comment:
I would not remove the call to the attestation script, i would make it conditional instead, the default behavior can be what you want (no setup of attestation) but if user want, they can have enable attestation setup just by setting a variable, it will also make the README shorter

Reply:
Regarding making the README shorter by adding the auto install via the setup-tdx-host.sh script: there's really not that much savings.  Right now (manual way), it adds 6 lines of text.  If i have to tell them to create an environment variable, etc. I think it will be almost the same amount of text to explain.  

And could be more prone to problems.  For example, If they missed (didn't read carefully, as users often do) the instruction to create the environment variable to add DCAP as part of the setup-host-tdx.sh script and when they get to step 8 to setup attestation, they don't have DCAP.  how would they know to install DCAP? I'll have to add extra instruction that installing DCAP is necessary before proceeding with the configuration and if you didn't install DCAP, here are the steps, etc.  

IMO, documenting exact steps to install DCAP is less error prone. 

What do you think?  :)

Reply:
If they missed, they can re-run the script again, from my perspective, i would like to have a way to have a full setup at once

Reply:
ok. Made changes per your preference.